cmd/kube-proxy/app/server_windows.go

```diff
@@ -183,6 +185,16 @@ func newProxyServer(config *proxyconfigapi.KubeProxyConfiguration, cleanupAndExi
 	}, nil
 }
 
+func getDualStackMode(networkname string, compatTester winkernel.StackCompatTester) bool {
+	dualstack, err := compatTester.DualStackCompatible(networkname)
+	if err != nil {
+		klog.Errorf("unable to determine dual-stack status %v. Falling back to single-stack", err)
+		return dualstack
```

Comment:
The `return dualstack` outside the `err != nil` check should be sufficient

Reply:
good point, the logging was inconsistent here (some was here some down in the function itself) so I moved it all to one location